### PR TITLE
Fix Issue #81:  Docstring to info.py - calc

### PR DESCRIPTION
**Pull request by OBITORASU**
## Description
Added the ```info.py``` to the calc folder aimed at turning the docstring in ```main.py``` into the file itself.

Fixes Issue #81

### CODE REVIEW

File: honeybot/plugins/downloaded/calc/info.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+NAME = 'abbreviation.py'
```
Comment (Abdur-rahmaanJ) [inline honeybot/plugins/downloaded/calc/info.py:2]: The name should be calc i guess?

Comment (OBITORASU) [reply honeybot/plugins/downloaded/calc/info.py:2]: Yes missed this one my bad.

**Comment by Abdur-rahmaanJ** [comment]
Great! Merging!

**Comment by Abdur-rahmaanJ** [comment]
Feel free to join the discord btw

**Pull request merged**